mesonbuild/dependencies/mpi.py

```diff
@@ -196,6 +207,33 @@ def _sanitize_version(self, out: str) -> str:
         return out
 
 
+class MSMPIConfigToolDependency(_MPIConfigToolDependency):
```

Comment:
Does it really make sense to inherit the `_MPIConfigToolDependency` here? It seems like you're just working around it. maybe it would make more sense to just inherit `ConfigToolDependency` directly?

Reply:
yes